packages/common/src/sites/_internal/_migrate-telemetry-config.ts

```diff
@@ -0,0 +1,59 @@
+import { getProp, setProp } from "../../objects";
+import { IModel, IDraft } from "../../types";
+import { cloneObject } from "../../util";
+
+/**
+ * Reconfigure event list card properties
+ * @private
+ * @param {object} model Site Model
+ * @returns {object}
+ */
+export function _migrateTelemetryConfig<T extends IModel | IDraft>(
+  model: T
+): T {
+  const newSchemaVersion = 1.7;
+  // do nothing if migration already applied
+  if (getProp(model, "item.properties.schemaVersion") >= newSchemaVersion) {
+    return model;
+  }
+
+  // apply migration
+  const clone = cloneObject(model);
+
+  clone.data.telemetry = {};
+  // get allowPrivacyConfig from consentNotice capability
+  const capabilities = getProp(model, "data.values.capabilities") || [];
```

Comment:
wondering if we should convert this capability to a permissions system feature like we recently did for the "hideFollow" and "disableDiscussions" capabilities? 

Essentially you'd need to: 
1. add a new feature flag to the `SiteDefaultFeatures` [here](https://github.com/Esri/hub.js/blob/feat/7735-advanced-json-editor/packages/common/src/sites/_internal/SiteBusinessRules.ts#L10-L11)
2. add a mapping for the capability to feature [here](https://github.com/Esri/hub.js/blob/feat/7735-advanced-json-editor/packages/common/src/sites/_internal/capabilities/capabilityToFeatureMap.ts)

The `migrateLegacyCapabilitiesToFeatures` and `convertFeaturesToLegacyCapabilites` functions should handle the rest

Then here, you'd reference `getProp(model, "data.settings.features")` instead

Reply:
Well, this is something we hope to be used across the platform so i think we really want it to be at data.telemetry.consentNotice.allowPrivacyConfig

Reply:
Additionally, this config is passed into telemetry.js, and everything else is handled internally - the app does not need to know anything about the privacy settings in terms of permissions